.github/actions/update-data/dist/index.mjs

```diff
@@ -9319,31 +9319,12 @@ async function updateData() {
 
   const cardChanges = await compareFile(changes, 'cards.js', getCardsJs);
   if (cardChanges) {
-    const spoilersJs = await getSpoilersJs(cardChanges);
-    changes.push({
-      folder: 'scripts/data/',
-      filename: 'spoilers.js',
-      contents: spoilersJs
-    });
+    await compareFile(changes, 'spoilers.js', getSpoilersJs, cardChanges);
```

Comment:
🗒️ Use the `.github/actions/build.sh` script to bundle the actions.  There are a few issues with the bundler that need to be corrected by the script.